README.rst

```diff
@@ -50,6 +53,12 @@ Or if you don't need the ROS bindings:
 
   pip install dual_quaternions
 
+ROS package
+~~~~~~~~~~~
+
+Release into apt is on its way. Until then you'll have to build the catkin package from source. Simply clone this repo into your
+catkin workspace and build with :code:`catkin_make`
```

Comment:
for totally new users, it would be useful to say that they should clone it into their src file. For ex: 
```bash
cd ~/catkin_ws/src
git clone https://github.com/Achllle/dual_quaternions_ros
cd ..
catkin_make
```

Reply:
done